nestkernel/nest_time.cpp

```diff
@@ -77,7 +78,7 @@ Time::compute_max()
   const tic_t tmax = std::numeric_limits< tic_t >::max();
 
   tic_t tics;
-  if ( lmax < tmax / Range::TICS_PER_STEP ) // step size is limiting factor
+  if ( lmax < tmax * Range::TICS_PER_STEP_INV ) // step size is limiting factor
```

Comment:
@heplesser This is also a question of taste. On the one hand, not changing anything that doesn't need to be changed is "clearer" since we have a minimal change; on the other hand, using all * instead of / is also "clearer".